Makefile

```diff
@@ -620,6 +620,7 @@ edpm_deploy_baremetal_prep: export EDPM_BMH_NAMESPACE=${BMH_NAMESPACE}
 edpm_deploy_baremetal_prep: export EDPM_PROVISIONING_INTERFACE=${BMO_PROVISIONING_INTERFACE}
 edpm_deploy_baremetal_prep: export EDPM_CTLPLANE_INTERFACE=${BMO_CTLPLANE_INTERFACE}
 edpm_deploy_baremetal_prep: export EDPM_TOTAL_NODES=${DATAPLANE_TOTAL_NODES}
+edpm_deploy_baremetal_prep: export OPENSTACK_RUNNER_IMG=${DATAPLANE_RUNNER_IMG}
```

Comment:
This won't reflect for default services as they're reconciled by dataplane-operator. Only the custom services in devsetup/edpm/services would be kustomized before being created.